cli/lib/status.ts

```diff
@@ -34,11 +34,28 @@ type QueryResponse = {
 };
 
 export const getStatus = async (network: string) => {
-  const response = await request<QueryResponse>(
-    "https://api.thegraph.com/index-node/graphql",
-    query,
-    { subgraph: `requestnetwork/request-payments-${network}` },
-  );
+  let response;
```

Comment:
just a detail, but you could instead have a mapping for the root URL, based on the network key: 

```js
const graphNode = {
 "mantle-testnet": "",
 "mantle": ""
}
const nodeUrl = graphNode[network] || defaultUrl;
```

Reply:
@benjlevesque What do you think of this? https://github.com/RequestNetwork/payments-subgraph/pull/72/commits/326f25c309b843c899c64c73a7d7ee8e552c9035

I don't have time to finish and test it today so I moved the PR back to draft status.

Reply:
looks good yes